Example/RaisinToast/RZTAppDelegate.m

```diff
@@ -19,13 +19,17 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
 {
     [self.window makeKeyAndVisible];
     // Override point for customization after application launch.
-    self.errorWindow = [RZMessagingWindow messagingWindow];
-    [RZErrorMessenger setDefaultMessagingWindow:self.errorWindow];
-    [RZErrorMessenger setDefaultErrorDomain:[NSString stringWithFormat:@"%@.error",[[NSBundle mainBundle] bundleIdentifier]]];
```

Comment:
Any reason the default error domain line didn't make it over to the didBecomeActive implementation?